lib/util/log.js

```diff
@@ -57,6 +59,29 @@ class Log {
       _warnings: {
         value: new Set()
       },
+      _loggerName: {
+        get: function get() {
+          return loggerName && typeof loggerName === 'string' ? loggerName : DEFAULT_LOGGER_NAME;
+        }
+      },
+      _logger: {
+        get: function get() {
+          const logger = getLogger(this._loggerName);
+          let level = this._logLevels[moduleName] || DEFAULT_LOG_LEVEL;
+
+          // There is no 'off' in the logger module. It uses 'silent' instead
+          level = level === 'off' ? 'silent' : level;
+
+          logger.setDefaultLevel(level);
+          return logger;
+        }
+      },
+      _logLevelsEqual: {
```

Comment:
I think we do not need to keep old logger. we should either 
a) remove support for logLevels object - which allows different log levels for different modules
b) use different loggerName for different modules. that way we can keep the logLevels object support but still use "loglevel" library.
for example all logs for a given modulename will go in: 
var moduleLogger = require("loglevel").getLogger(this._loggerName + "-" + moduleName);

Reply:
Per our slack discussion, we'll go with b